HISTORY.md

```diff
@@ -1,5 +1,11 @@
 # History
 
+## 1.2.4
```

Comment:
please remove that from history, I'll add things in the release commit, when I release

Reply:
Yes I do it